cpmonitor/templates/circle-checked-filled.html

```diff
@@ -0,0 +1,14 @@
+<svg style="color:green"
```

Comment:
@kleins05 Do you have colors that I should use or are these ok?

Reply:
@cafischer I think it'd be nice to re-use the colors that I use for the statuses, the css default colors are too saturated for my taste. The status colors are defined in main.scss in class lz-status. If we now need them in other places, too, we should probably define css variables. 
Also, if possible, please try to avoid inline styles and use classes instead. I am surprised djlint did not pick up on it. With [rule H021](https://www.djlint.com/docs/linter/) back in place since #164 it should complain. Also if we want to enforce a [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) at some point, we should try to work without inline styles.

Reply:
Weird that djlint did not complain, do we need to investigate?

Reply:
I created an issue for djlint #170

Reply:
I hope I did what you wanted regarding the colors, otherwise just make a commit.